common/onepanel/base/secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: onepanel
+  namespace: $(defaultNamespace)
+type: Opaque
+stringData:
+  argoArtifactRepositoryS3Bucket: $(artifactRepositoryBucket)
+  argoArtifactRepositoryS3KeyFormat: artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
+  argoArtifactRepositoryS3Endpoint: $(artifactRepositoryEndpoint)
+  argoArtifactRepositoryS3Insecure: $(artifactRepositoryInsecure)
+  argoArtifactRepositoryS3AccessKeySecretName: $(artifactRepositoryAccessKeySecretName)
+  argoArtifactRepositoryS3AccessKeySecretKey: $(artifactRepositoryAccessKeySecretKey)
```

Comment:
I think this is supposed to the key in the secret, not `$(artifactRepositoryAccessKeySecretKey)`. This also incorrect in `workflow-controller-parameters` (is this configMap even needed?)